### ISSUE TITLE
REM sleep behavior disorder  -> Dream enactment behavior

### ISSUE THREAD

**Issue by SmartMonkey-git** [obophenotype/human-phenotype-ontology#11459]
**HPO term**
REM sleep behavior disorder HP:5200291

**Synonym to be added**
Dream enactment behavior

**broad/exact/narrow/plural/abbreviation ?**
exact

**Source**
Found the term in the book `This Is Why You Dream` by Rahul Jandial and this [paper](https://pmc.ncbi.nlm.nih.gov/articles/PMC8034224/), which also offers a definition of the term.

Maybe, think of adding both British and American spelling of the word behavior? The synonyms of the term seem mixed.

**Comment by pnrobinson** [comment]
@SmartMonkey-git I think this may require some thought. Dream Enactment Behavior is a feature of REM sleep behavior disorder and can also occur in other situations. I am not sure that REM sleep behavior disorder is a great HPO term since it is a bundled diagnosis

**Comment by pnrobinson** [comment]
Reading the article, I think we should relabel the term [REM sleep behavior disorder HP:5200291](https://hpo.jax.org/browse/term/HP:5200291) -- because the latter is a disease and Dream enactment behavior may occur in other settings.

Label: **Dream enactment behavior**
Def: The acting out of dreams, characterized by body movement, emotional expression, or audible verbalization of dream content has been termed dream enactment behavior (DEB) when the body’s physical expression is followed by the individual recalling a dream. 
Comment: Dream enactment behavior can range in intensity and frequency in the normal population. In most cases, DEB occurs during the transition from sleep to wake, which allows the individual to recall the event coincident with dream imagery immediately after it took place. This is unlike typical somnambulism or somniloquy, which generally occur during non-REM (NREM) sleep without any dream recall. Dream enactment behavior is the defining clinical feature of rapid eye movement (REM) sleep behavior disorder (RBD). Patients are reported by their bed partners to scream, speak, fall, or move their arms and legs to mimic their dreams.
Synonym: REM sleep behavior disorder
